From 2babecb35db3b1f605f50f89e1384940b14317f7 Mon Sep 17 00:00:00 2001
From: Srujana Challa <schalla@marvell.com>
Date: Wed, 25 Nov 2020 14:22:11 +0530
Subject: [PATCH 0942/1921] crypto: marvell: fix a missing Kconfig dependency

This patch enables OCTEONTX2_MBOX to be built for CPT
in Kconfig as it is required to build CPT.

Signed-off-by: Srujana Challa <schalla@marvell.com>
Change-Id: I12d1854eab0e3691a033cea5ddcca11744f9a519
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/40963
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/crypto/marvell/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/crypto/marvell/Kconfig b/drivers/crypto/marvell/Kconfig
index c059cdf90437..f608eb509a5b 100644
--- a/drivers/crypto/marvell/Kconfig
+++ b/drivers/crypto/marvell/Kconfig
@@ -41,6 +41,7 @@ config CRYPTO_DEV_OCTEONTX2_CPT
 	depends on ARCH_THUNDER || COMPILE_TEST
 	depends on PCI_MSI && 64BIT
 	depends on CRYPTO_LIB_AES
+	select OCTEONTX2_MBOX
 	select CRYPTO_SKCIPHER
 	select CRYPTO_HASH
 	select CRYPTO_AEAD
@@ -57,6 +58,7 @@ config CRYPTO_DEV_CN10K_CPT
 	depends on ARCH_THUNDER || COMPILE_TEST
 	depends on PCI_MSI && 64BIT
 	depends on CRYPTO_LIB_AES
+	select OCTEONTX2_MBOX
 	select CRYPTO_DEV_MARVELL
 	select CRYPTO_SKCIPHER
 	select CRYPTO_HASH
-- 
2.31.1

